package.json

```diff
@@ -64,13 +74,14 @@
   "devDependencies": {
     "babel-runtime": "5.8.29",
     "babelify": "6.4.0",
-    "browserify": "6.1.0",
+    "browserify": "10.0.0",
+    "budo": "^8.2.1",
     "chai": "1.9.2",
     "chai-as-promised": "4.1.1",
-    "derequire": "2.0.3",
```

Comment:
There was a reason we were using `derequire`, but I don't remember what it is. @louh should know (I believe he added it).